tests/phpunit/integration/Core/Modules/ModuleTest.php

```diff
@@ -70,7 +70,6 @@ public function test_prepare_info_for_js() {
 			'cta',
 			'sort',
 			'homepage',
-			'learnMore',
```

Comment:
Please, revert `learnMore` removal from test files since it hasn't been required.

Reply:
👍